### DISCUSSION TITLE
Artisan PID and SSR

### DISCUSSION THREAD

**Discussion by madwheeling** [artisan-roaster-scope/artisan#606] [Q&A] [answered]
Is there a way to get a lower PWM frequency to control a zero crossing SSR and still use the built in PID? If not what is the best alternative? I think I would have to get an expensive proportional SSR or an external PID for my setup to work.

**Comment by MAKOMO** [comment, marked as answer]
Artisan itself is not producing PWM signals. Instead it relies on connected hardware for this. Artisan supports for example the `Yoctopuce PWM-Tx` which allows to set the frequency and duty separately. 

https://artisan-scope.org/devices/yoctopuce/

Not sure if this serves your purpose.
